Cargo.toml

```diff
@@ -15,18 +15,24 @@ description = "Packaging tools for Oxide's control plane software"
 [dependencies]
 anyhow = "1.0"
 async-trait = "0.1.67"
+blake3 = { version = "1.5", features = ["mmap", "rayon"] }
+camino = { version = "1.1", features = ["serde1"] }
+camino-tempfile = "1.1"
 chrono = "0.4.24"
 filetime = "0.2"
 flate2 = "1.0.25"
+futures = "0.3"
 futures-util = "0.3"
 hex = "0.4.3"
+once_cell = "1.19"
```

Comment:
I believe you don't need to pull in the crate for this; https://doc.rust-lang.org/std/sync/struct.Once.html#method.call_once should handle the use case here. I could be wrong though!

Reply:
I had to use the `std::sync::OnceLock`, but it appears to be a drop-in replacement.

Done in 3ff7174af60d12a43a266114079e01beb2269585